frontend/public/components/job.tsx

```diff
@@ -232,6 +194,47 @@ const JobsList: React.FC = (props) => {
       props: { className: tableColumnClasses[5] },
     },
   ];
+  const JobTableRow: RowFunction<JobKind> = ({ obj: job, index, key, style }) => {
+    const { type, completions } = getJobTypeAndCompletions(job);
+    return (
+      <TableRow id={job.metadata.uid} index={index} trKey={key} style={style}>
+        <TableData className={tableColumnClasses[0]}>
+          <ResourceLink
+            kind={kind}
+            name={job.metadata.name}
+            namespace={job.metadata.namespace}
+            title={job.metadata.uid}
+          />
+        </TableData>
+        <TableData
+          className={classNames(tableColumnClasses[1], 'co-break-word')}
+          columnID="namespace"
+        >
+          <ResourceLink
+            kind="Namespace"
+            name={job.metadata.namespace}
+            title={job.metadata.namespace}
+          />
+        </TableData>
+        <TableData className={tableColumnClasses[2]}>
+          <LabelList kind={kind} labels={job.metadata.labels} />
+        </TableData>
+        <TableData className={tableColumnClasses[3]}>
+          <Link
+            to={`/k8s/ns/${job.metadata.namespace}/jobs/${job.metadata.name}/pods`}
+            title="pods"
+          >
+            <div>{t('workload~Desired {{completions}}', { completions })}</div>
+            <div>{t('workload~Succeeded {{count}}', { count: job.status.succeeded || 0 })}</div>
```

Comment:
My bad.  `count` is a reserved keyword for pluralization.  Use `succeeded` as the key instead.